core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -147,7 +147,9 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
   protected def closeSockets(): Unit = {
     while (openSockets.nonEmpty) {
       val socket = openSockets.head
-      socket.close()
+      if (!socket.isClosed) {
+        socket.close()
+      }
```

Comment:
Seems strange that we are fetching from `openSockets` and checking if it's closed. So if it was `closed` then why not removed.